### ISSUE TITLE
Missing required key 'Item' in params

### ISSUE THREAD

**Issue by markwylde** [KoryNunn/alternator#3]
Hey Kory

Was trying to use your library to connect to dynamodb, but getting the error below:

**My Code:**
```javascript
const AWS = require('aws-sdk')
const alternator = require('alternator')

const db = alternator({
  endpoint: new AWS.Endpoint('https://dynamodb.eu-west-1.amazonaws.com'),
  accessKeyId: 'xxx',
  secretAccessKey: 'xxx',
  region: 'eu-west-1'
},

[{
  name: 'users',
  key: {
    name: 'hash',
    version: 'range'
  },
  attributes: {
    name: 'string',
    version: 'number'
  }
}])

// Create a user
db.table('users').create({
  name: 'bob',
  version: 0
}, console.log)
```

**Error:**
```bash
MissingRequiredParameter: Missing required key 'Item' in params
    at ParamValidator.fail (/Users/mark/Documents/dyna/node_modules/aws-sdk/lib/param_validator.js:50:37)
    at ParamValidator.validateStructure (/Users/mark/Documents/dyna/node_modules/aws-sdk/lib/param_validator.js:61:14)
    at ParamValidator.validateMember (/Users/mark/Documents/dyna/node_modules/aws-sdk/lib/param_validator.js:88:21)
    at ParamValidator.validate (/Users/mark/Documents/dyna/node_modules/aws-sdk/lib/param_validator.js:34:10)
    at Request.VALIDATE_PARAMETERS (/Users/mark/Documents/dyna/node_modules/aws-sdk/lib/event_listeners.js:126:42)
    at Request.callListeners (/Users/mark/Documents/dyna/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at callNextListener (/Users/mark/Documents/dyna/node_modules/aws-sdk/lib/sequential_executor.js:96:12)
    at /Users/mark/Documents/dyna/node_modules/aws-sdk/lib/event_listeners.js:86:9
    at finish (/Users/mark/Documents/dyna/node_modules/aws-sdk/lib/config.js:349:7)
    at /Users/mark/Documents/dyna/node_modules/aws-sdk/lib/config.js:367:9 {
  message: "Missing required key 'Item' in params",
  code: 'MissingRequiredParameter',
  time: 2019-07-23T08:51:21.949Z
} false
```

Any ideas what I'm doing wrong?

The get seems to work perfectly if I just manually create 'bob' in the AWS attempt at a UI :S

```javascript
// Get a user
db.table('users').get({
    key: {
        name: 'bob',
        version: 0
    }
}, callback);
```

```bash
null { name: 'bob', version: 23, test: 'this is a test' }
```

**Comment by KoryNunn** [comment]
My bad, I've fixed the README: https://github.com/KoryNunn/alternator/commit/5d5506ea5419ba61c08fddbbe460acbe3aa3eaf3

tl;dr:

```
db.table('users').create({
  item: {
  	name: 'bob',
  	version: 0
  }
}, console.log)
```
